tools/ci/run_tc.py

```diff
@@ -211,10 +211,10 @@ def download_artifacts(artifacts):
         print("DEBUG: Got artifacts %s" % artifacts_data)
```

Comment:
Any idea why this wasn't rewritten?

Reply:
Based on a more thorough investigation, I think the pyupgrade script has some things it won't handle.
It will use the `.format()` syntax or it won't change percent format:
1. in a print statement (sometimes?)
2. if a multi-line string is concatenated while also using formatting, e.g.
```
("INFO: Merge commit changed from {} to {} "
 "due to base branch changes. Running task anyway.") % (var1, var2)
```
3. if the variables added to the string are accessed from a dictionary using the same type of quotes, e.g.
`"Code is: {}" % dict["code"]` will not be fixed because the "code" key is being accessed using the same double quotes as the string.